vivisect/symboliks/common.py

```diff
@@ -249,15 +249,19 @@ def setSymKid(self, idx, kid):
             todo = collections.OrderedDict({p._sym_id: p for p in oldkid.parents})
             done = set()
             while todo:
-                pid, parent = todo.popitem()
+                pid, parent = todo.popitem(last=False)
```

Comment:
does first/last make a significant difference?

Reply:
No, so I removed this.